CONTRIBUTING.md

```diff
@@ -60,19 +62,19 @@ cargo xtask gen-receipt
 Before submitting a PR, ensure the following:
 
 1. Fork the `risc0` repository and create a new branch there to do your work.
-1. Format with `cargo fmt --all`.
-1. Lint with:
-    ```
-    RISC0_SKIP_BUILD=1 cargo clippy
-    $(cd examples && RISC0_SKIP_BUILD=1 cargo clippy)
-    $(cd benchmarks && RISC0_SKIP_BUILD=1 cargo clippy)
-    ```
-1. Perform the license check with `python3 license-check.py`.
-1. Tests pass with:
-    ```
-    cargo test -F prove -F docker
-    $(cd examples && cargo test)
-    $(cd benchmarks && cargo test)
-    ```
-1. Commit any outdated `Cargo.lock` files.
-1. Open a PR against the `main` branch.
+2. Format with `cargo fmt --all`.
+3. Lint with:
```

Comment:
These should auto-number when it renders, so leaving `1.` should work

Reply:
This was from running the preferred formatter for docs on all the md I touched: 

https://github.com/risc0/risc0/blob/f3a78d6f4babac2106e64e3d731f348b75735fee/website/package.json#L16-L18
https://github.com/risc0/risc0/blob/f3a78d6f4babac2106e64e3d731f348b75735fee/website/.remarkrc#L39-L50

I know that manual numbering is the strong preference for @nategraf that might wanna chime in here in gerenal (we can make a new issue if we feel being uniform in how to use md ordered lists is worthwhile)

![image](https://github.com/user-attachments/assets/f178b28c-1905-451a-8937-14bb1bce827c)

Reply:
OK, not a huge deal either way